src/main/java/carpet/mixins/Level_tickMixin.java

```diff
@@ -19,14 +20,20 @@
 import net.minecraft.world.level.Level;
 
 @Mixin(Level.class)
-public abstract class Level_tickMixin implements WorldInterface
+public abstract class Level_tickMixin implements LevelInterface
 {
     @Shadow @Final public boolean isClientSide;
+    @Shadow @Final protected NeighborUpdater neighborUpdater;
     CarpetProfiler.ProfilerToken currentSection;
     CarpetProfiler.ProfilerToken entitySection;
 
     Map<EntityType<?>, Entity> precookedMobs = new HashMap<>();
 
+    @Override
+    public NeighborUpdater getNeighborUpdater() {
+        return this.neighborUpdater;
+    }
```

Comment:
```suggestion
    @Override
    @Unique
    public NeighborUpdater getNeighborUpdater() {
        return this.neighborUpdater;
    }
```
I think Unique can/should go there, if so why not, may save us the mixin if mojang adds it some day.

Needs an import.